.gitignore

```diff
@@ -1,3 +1,7 @@
-/roles
+playbooks/roles/*
+!playbooks/roles/consul
+!playbooks/roles/node-exporter
+!playbooks/roles/ernestas-poskus.prometheus
```

Comment:
Yep, eventually we'll move everything to submodules, which'll allow all roles to be in this repo at all times. It's the cleanest way to avoid a hybrid situation.